src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/test_hdinsight_commands.py

```diff
@@ -47,6 +47,16 @@ def test_hdinsight_cluster_kafka(self, storage_account_info):
             HDInsightClusterTests._kafka_arguments()
         )
 
+    # Uses 'rg' kwarg
+    @ResourceGroupPreparer(name_prefix='hdicli-', location=location, random_name_length=12)
+    @StorageAccountPreparer(name_prefix='hdicli', location=location, parameter_name='storage_account')
+    def test_hdinsight_cluster_kafka_with_rest_proxy(self, storage_account_info):
+        self._create_hdinsight_cluster(
```

Comment:
Typically we will use test scenarios for commands. Can you add test for your new parameter with self.cmd here?

Reply:
> Typically we will use test scenarios for commands. Can you add test for your new parameter with self.cmd here?

Hi @Juliehzl , Thanks for your comment. We really cover the self.cmd lever.
The self._create_hdinsight_cluster will actually call `az hdinsight create` cmd. 
The definition of `_create_hdinsight_cluster` is at [here](https://github.com/Azure/azure-cli/blob/22035a769df76a48d964477f1c0e0e63ed6c174a/src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/test_hdinsight_commands.py#L297)

And from this [line](https://github.com/Azure/azure-cli/blob/22035a769df76a48d964477f1c0e0e63ed6c174a/src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/test_hdinsight_commands.py#L305) we can see that this method wil call `az hdinsight create` cmd.

Thanks